vessel_scoring/nnet_model.py

```diff
@@ -0,0 +1,513 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import tensorflow as tf
+import numpy as np
+import time
+
+def get_polynomial_cols(x, windows):
+    colnames = []
+    #colnames.append("speed")
+    for window in windows:
+        colnames.append('measure_speedavg_%s' % window)
+        colnames.append('measure_speedstddev_%s_log' % window)
+        colnames.append('measure_coursestddev_%s_log' % window)
+    cols = [x[col] for col in colnames]
+    return cols
+
+
+
+# Some of this code comes from Google Tensor flow demo: check license
+# and license this appropriately or rewrite from scratch.
```

Comment:
The code is Apache II, so you should paste in the copyright notice in this code, but then that's it and we can use it. This is what you need to paste.

# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
# ==============================================================================